experimental/FTheoryTools/src/LiteratureModels/constructors.jl

```diff
@@ -313,6 +313,8 @@ function literature_model(model_dict::Dict{String, Any}; model_parameters::Dict{
     
     for (key, val) in model_parameters
       map!(x -> nested_string_map(s -> replace(s, key => string(val)), x), values(model_dict["model_data"]))
+      map!(x -> nested_string_map(s -> replace(s, key => string(val)), x), values(model_dict["model_descriptors"]))
+      map!(x -> nested_string_map(s -> replace(s, r"\(([^)]+)\)" => dim -> string("(", Oscar.eval_poly(string.(match(r"\(([^)]+)\)", dim).captures[1]), polynomial_ring(ZZ, Symbol[])[1]),")")), x), values(model_dict["model_descriptors"]))
```

Comment:
```suggestion
      map!(x -> nested_string_map(s -> replace(s, r"\(([^)]+)\)" => dim -> string("(", Oscar.eval_poly(string.(match(r"\(([^)]+)\)", dim).captures[1]), ZZ),")")), x), values(model_dict["model_descriptors"]))
```
It noticed that one can actually do `Oscar.eval_poly("2*2+1", ZZ)`.